source/content/guides/composer-convert.md

```diff
@@ -167,77 +312,54 @@ If you are using an exported config, you will need to move the configuration fil
+You've now committed the code to the local branch. Deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize the site's logs via `terminus drush $SITE.composerify -- wd-show` to inspect the watchdog logs, or follow the directions in our documentation on [log collection](/logs).
 
-If you do plan on using a CI solution, refer to our [Build Tools](/guides/build-tools) guide at this point.
+### Deploy to a Multidev
 
-From the `$site-composer` directory, run the following:
+Push the changes to a Multidev called `composerify` to safely test the site without affecting the Dev environment:
 
 ```bash{promptUser:user}
-composer prepare-for-pantheon
-composer install --no-dev
-```
-
-This should modify the `.gitignore` file and cleanup any errant `.git` directories in the codebase, to prepare your new code for direct deployment to Pantheon.
-
-## Commit
-
-Commit your work to the git repo. From the `$site` directory, run the following:
-
-```bash{promptUser: user}
-cp -r .git ../$site-composer/
-cd ../$site-composer
 git add .
-git commit -m "Convert to Composer based install"
+git commit -m "ran composer prepare-for-pantheon and install"
+git push origin composerify && terminus env:create $SITE.dev composerify
 ```
 
-You should see a large amount of files committed to the new branch we created earlier.
+Once you have confirmed the site is working, merge `composerify` into `master`, and follow the standard workflow to QA a code change before going live.
 
-## Deploy
+### Deploy to Dev
 
-You've now committed the code to a branch. If your site has Multidev, you can deploy that branch directly to a new Multidev and test the site in the browser. If the site doesn't load properly, clear the cache. If there are any issues, utilize your site's logs via `terminus drush $site.composify -- wd-show` to inspect the watchdog logs, or follow the directions on our documentation on [log collection](/logs).
+If you have a [local development](/local-development) solution, consider testing your `composerify` branch locally before merging.
 
-Once you have confirmed the site is working, merge `composify` into `master`, and follow the standard workflow to QA a code change before going live.
+To force push the changes from the local branch to the `master` Dev branch:
 
-If your plan does not include Multidev, you will have to merge to master before deploying, then follow the rest of the steps above. If you have a local development solution, consider testing your `composify` branch locally before merging.
+```bash{promptUser:user}
+git add .
+git commit -m "ran composer prepare-for-pantheon and install"
```

Comment:
Comment does not match instructions; `composer prepare-for-pantheon` isn't part of this guide (this is from the guild on converting a site to build tools)